examples/eepromfstest/eepromfstest.c

```diff
@@ -94,7 +94,7 @@ static void print_game_high_scores(void)
 static int read_game_high_scores(char * path)
 {
     printf( "Reading '%s'\n", path );
-    const int result = eepfs_read(path, &game_high_scores);
+    const int result = eepfs_read(path, &game_high_scores, sizeof(game_high_scores));
```

Comment:
This is an unrelated change, I think it should not be part of this PR

Reply:
It's a minor change; I included it because the example was broken, and I needed to test each example's Makefile. The size parameter of of eepfs_read and eepfs_write was a last-second addition to the PR that introduced this example.